aii-ks/src/main/perl/ks.pm

```diff
@@ -73,9 +68,16 @@ use constant { KS               => "/system/aii/osinstall/ks",
                INIT_SPMA_IGN_DEPS   => "/system/aii/osinstall/ks/init_spma_ignore_deps",
            };
 
+# These attributes are not configuration options
+# profile is handled separately
+use constant CCM_CONFIG_NOOPTIONS => qw(
```

Comment:
This is brittle to the component schema changing and introducing new options.  There's also the possibility of breaking your ccm if you specify group_readable but the group isn't available during the install. I would prefer a whitelist of known config options, and default owner-only.